editor/filesystem_dock.cpp

```diff
@@ -2749,6 +2763,17 @@ FileSystemDock::FileSystemDock(EditorNode *p_editor) {
 	file_list_display_mode = FILE_LIST_DISPLAY_THUMBNAILS;
 
 	always_show_folders = false;
+
+	shell_ignored_extensions.push_back("scn");
```

Comment:
Couldn't we compute that automatically? Or maybe get it from somewhere? 
I am pretty sure we will forget to maintain this list if we makes changes somewhere.

Reply:
Any suggestions how to do this automatically? Alternatively we can do it by whitelist.